plenum/persistence/req_id_to_txn.py

```diff
@@ -11,30 +11,36 @@ class ReqIdrToTxn:
     """
 
     def __init__(self, keyValueStorage: KeyValueStorage):
```

Comment:
Do we have unit tests for `ReqIdrToTxn`?

Reply:
We have only integration test in test_already_processed_requests(). Should to add unit tests too?

Reply:
Done.